source/user/mcscf.rst

```diff
@@ -1,57 +1,299 @@
+  mycas.kernel()
+
+See :source:`examples/mcscf/41-state_average.py` for a complete example.
+
+
+Job Control
+-----------
+
+Optimization Settings
+"""""""""""""""""""""
+
+For CASSCF calculations, users may want to modify several of the convergence tolerances such as the energy tolerance (``conv_tol``), the orbital gradient tolerance (``conv_tol_grad``), and the maximum number of MCSCF iteration (``max_cycle_macro``).
+
+.. code-block:: python
+
+  mycas = mcscf.CASSCF(mf, 6, 6)
+  mycas.conv_tol = 1e-12
+  mycas.conv_tol_grad = 1e-6
+  mycas.max_cycle_macro = 25
+  mycas.kernel()
+
+
+Initial Guess
+"""""""""""""
+
+Initial guess orbitals for the CASSCF calculation may be passed to the ``kernel`` member function of an MCSCF object.
+
+.. code-block:: python
+
+  mycas = mcscf.CASSCF(myhf, 8, 8)
+  mycas.kernel(my_custom_mos)
+
+
+CI coefficient from previous or related calculations can also be passed as an initial guess to expedite a calculation:
+
+.. code-block:: python
+  mycas = mcscf.CASSCF(myhf, 8, 8)
+  mycas.kernel(my_custom_mos, my_custom_ci)
+
+Examples:
+
+* :source:`examples/mcscf/14-project_init_guess.py`
+* :source:`examples/mcscf/31-cr2_scan/cr2-scan.py`
+* :source:`examples/mcscf/34-init_guess_localization.py`
+* :source:`examples/mcscf/43-avas.py`
+* :source:`examples/mcscf/43-dmet_cas.py`
+
+
+Restarting
+""""""""""
+
+.. warning::
+  When running large calculations, it's always recommended that you specify a checkpoint file for your calculation.
+
+.. code-block:: python
+
+  mycas.chkfile = "casscf.chk"
+
+Much like :mod:`scf`, if a job is interrupted, users can restart the MCSCF calculations using checkpoint files from a crashed calculation.
+
+.. code-block:: python
+
+  from pyscf.lib import chkfile
+  old_chk_file = "old_casscf.chk"
+  mycas = mcscf.CASSCF(scf.RHF(mol), 6, 6)
+  mycas.chkfile = "restarted_casscf.chk"
+  mo = chkfile.load(old_chk_file, 'mcscf/mo_coeff')
+  mycas.kernel(mo)
+
+
+See :source:`examples/mcscf/13-restart.py` for a complete example.
+
+Observables and Properties
+--------------------------
+
+Wave Function Analysis
+""""""""""""""""""""""
+
+The ``analyze`` member functions of MCSCF objects prints many useful properties to ``stdout`` when the verbosity is >=4.
+
+1) Natural orbital occupancies
+2) Natural orbital AO expansions
+3) Overlap between canonical MCSCF orbitals and the initial guess orbitals.
+4) Analysis of the CI coefficients, i.e. the leading configurations and their weights
+5) AO populations
+6) Atomic populations
+7) AO spin densities (if applicable)
+8) Atomic spin densities (if applicable)
+
+.. code-block:: python
+  mycas = myhf.CASCI(6, 8).run()
+  mycas.verbose = 4
+  mycas.analyze()
+
+
+Natural Orbitals
+""""""""""""""""
+
+Users can request that the converged MCSCF orbitals be transformed with the member attribute ``natorb``
+
+.. warning::
+  When ``mycas.natorb`` is set, the natural orbitals may NOT be sorted by the active space occupancy.
```

Comment:
I wonder why this is; it sounds like the code should always sort the active space orbitals in occupation so that the active space window can be modified without problems...